### PR TITLE
NT-1997:UI/UX – Error loading more replies

### PR DESCRIPTION
**Pull request by hadia**
# 📲 What

handle problems loading more replies

# 🤔 Why

show pagination Error  loading extra pages of replies

# 🛠 How
same as comments we change the view more to the error message and its clickable to retry 

# 👀 See

https://user-images.githubusercontent.com/1075310/124611405-b5574c00-de71-11eb-8f8d-24a6f6c2b886.mp4

# 📋 QA

1. Test load replies  view more 
2. Produce pagination error "flight mode"
3. Click retry
4. try post replies

# Story 📖

https://kickstarter.atlassian.net/browse/NT-1997

### CODE REVIEW

File: app/src/main/java/com/kickstarter/viewmodels/ThreadViewModel.kt [hunk 1]
```diff
@@ -50,6 +50,9 @@ interface ThreadViewModel {
 
         fun isFetchingReplies(): Observable<Boolean>
         fun loadMoreReplies(): Observable<Void>
+
+        /** Display the bottom pagination Error Cell **/
```
Comment (Arkariang) [inline app/src/main/java/com/kickstarter/viewmodels/ThreadViewModel.kt:54]: On threads is displayed on top, modify the comment

File: app/src/main/java/com/kickstarter/viewmodels/ThreadViewModel.kt [hunk 2]
```diff
@@ -216,8 +231,30 @@ interface ThreadViewModel {
...
                 ?.share()
                 ?.subscribe {
+                    this.displayPaginationError.onNext(false)
                     this.onCommentReplies.onNext(it)
                 }
+
+            this.internalError
+                .compose(Transformers.combineLatestPair(onCommentReplies))
+                .filter {
+                    // it.first.second &&
```
Comment (Arkariang) [inline app/src/main/java/com/kickstarter/viewmodels/ThreadViewModel.kt:241]: Remove comment

**Review by Arkariang — APPROVED** [review]
- Some picky comments, but is good to go on my end :) 👍

File: app/src/main/java/com/kickstarter/viewmodels/ThreadViewModel.kt (hunk 2 above)
Comment (hadia) [reply app/src/main/java/com/kickstarter/viewmodels/ThreadViewModel.kt]: Done

File: app/src/main/java/com/kickstarter/viewmodels/ThreadViewModel.kt (hunk 1 above)
Comment (hadia) [reply app/src/main/java/com/kickstarter/viewmodels/ThreadViewModel.kt]: Done

**Pull request merged**